plugins/prestashop/prestashop_

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+: << =cut
+=head1 NAME
+
+Munin Plugin to grab PrestaShop statistics
+
+=head1 CONFIGURATION
+
+# Sample:
+[prestashop]
+	user root
+	env.database prestashop
+	env.dbprefix ps_
+
+=head1 AUTHOR
+
+# (c) 2015 Raphaël Droz <raphael.droz+floss@gmail.com>
+# General Public Licence v3 or later
+
+=head1 LICENSE
+
+GPLv3
+
+=head1 MAGIC MARKERS
+
+#%# family=auto
+#%# capabilities=autoconf suggest
+
+=cut
+
+graph_type=${0##*_}
+
+
+if [[ $1 = autoconf ]]; then
+    let ok=1
+    [[ -z $database ]] && echo "no (env.database not defined)" && ok=0
+    [[ -z $dbprefix ]] && echo "no (env.dbprefix not defined)" && ok=0
+    [[ -z $graph_type ]] && echo "no plugins linked with a bad name (unknown graph type '$graph_type')" && ok=0
+    [[ ! -r /etc/mysql/debian.cnf ]] && echo "no /etc/mysql/debian.cnf isn't readable" && ok=0
+    ! type -P mysql &>/dev/null && echo "no (one of mysql not found)" && ok=0
+    
+    (( $ok == 1 )) && echo yes
+    exit 0
+fi
+
+if [[ $1 = suggest ]]; then
+    for i in visits products orders ; do
+	echo $i
+    done
+    exit 0
+fi
+
+if [[ $1 = config ]]; then
+    if [[ $graph_type  == visits ]]; then
+	cat <<EOF
+graph_title PrestaShop connections
+graph_args --base 1000 -l 0
+graph_category php
```

Comment:
Please pick a category from the [list of commonly used ones](http://munin-monitoring.org/wiki/graph_category_list). For me "other" seems to be the most suitable one.